es_relval_log.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Notice: can't find usage of this script in cms jenkins

Reply:
`runPyRelValThread.py` (which should support both py2 and 3) makes use of it
```
runPyRelValThread.py:from es_relval_log import es_parse_log
```